Lib/test/test_ctypes/test_struct_fields.py

```diff
@@ -130,6 +130,21 @@ class S(Structure):
             self.check_struct(S)
             self.assertEqual(S.largeField.bit_size, size * 8)
 
+    def test_bitfield_overflow_error_message_gh143164(self):
```

Comment:
For simple and straightforward test cases, there is no need to mention the issue in the case name.

```suggestion
    def test_bitfield_overflow_error_message(self):
```